specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-01-01/costmanagement.json

```diff
@@ -0,0 +1,1438 @@
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/costmanagement/"
+        },
+        "x-ms-examples": {
+          "SubscriptionDimensionsList": {
+            "$ref": "./examples/SubscriptionDimensionsList.json"
+          },
+          "ResourceGroupDimensionsList": {
+            "$ref": "./examples/ResourceGroupDimensionsList.json"
+          },
+		  "ManagementGroupDimensionsList": {
+            "$ref": "./examples/ManagementGroupDimensionsList.json"
+          },
+          "ManagementGroupDimensionsListExpandAndTop": {
+            "$ref": "./examples/ManagementGroupDimensionsListExpandAndTop.json"
+          },
+          "ManagementGroupDimensionsListWithFilter": {
+            "$ref": "./examples/ManagementGroupDimensionsListWithFilter.json"
+          },
+          "DepartmentDimensionsList": {
+            "$ref": "./examples/DepartmentDimensionsList.json"
+          },
+          "DepartmentDimensionsListExpandAndTop": {
+            "$ref": "./examples/DepartmentDimensionsListExpandAndTop.json"
+          },
+          "DepartmentDimensionsListWithFilter": {
+            "$ref": "./examples/DepartmentDimensionsListWithFilter.json"
+          },
+          "EnrollmentAccountDimensionsList": {
+            "$ref": "./examples/EnrollmentAccountDimensionsList.json"
+          },
+          "EnrollmentAccountDimensionsListExpandAndTop": {
+            "$ref": "./examples/EnrollmentAccountDimensionsListExpandAndTop.json"
+          },
+          "EnrollmentAccountDimensionsListWithFilter": {
+            "$ref": "./examples/EnrollmentAccountDimensionsListWithFilter.json"
+          },
+          "BillingAccountDimensionsList": {
+            "$ref": "./examples/BillingAccountDimensionsList.json"
+          },
+          "BillingAccountDimensionsListExpandAndTop": {
+            "$ref": "./examples/BillingAccountDimensionsListExpandAndTop.json"
+          },
+          "BillingAccountDimensionsListWithFilter": {
+            "$ref": "./examples/BillingAccountDimensionsListWithFilter.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "scope",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The scope at which to list dimensions. This includes '/subscriptions/{subscriptionId}/' for subscription scope, '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}' for resourceGroup scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}' for Billing Account scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/departments/{departmentId}' for Department scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/enrollmentAccounts/{enrollmentAccountId}' for EnrollmentAccount scope and '/providers/Microsoft.Management/managementGroups/{managementGroupId}' for Management Group scope.",
+            "x-ms-skip-url-encoding": true
```

Comment:
use $ref approach and define it once to use at all other places.. 
{
            "$ref": "#/parameters/scopeParameter"
          },